docs/coding-standards/coding-style.md

```diff
@@ -0,0 +1,7 @@
+# Coding Style Guidelines
+
+1. _add an indentation example here_
+2. _add some naming examples (classes, ids, variables, functions)_
+3. _add some code comment examples/guidelines (eg, JSDoc)_
+4. Being consistent with surrounding code may trump the above guidelines!
+5. _add more here_
```

Comment:
Feel free to contribute screenshots and details for this PR... Or we can merge this PR now and you can add to this file in future PRs.